docs/_assets/style/custom.css

```
@@ -0,0 +1,68 @@
+  }
+}
+
+.g-root_theme_dark .pc-content-wrapper__icon[src*="github-dark"] {
+  filter: invert(1);
+}
+
+.pc-header-block__background {
+  border-radius: 20px !important;
+}
+
+/*FIX: page construct*/
```

Stylelint (17.12.0)

[error] Expected whitespace after "/*" (comment-whitespace-inside)

(comment-whitespace-inside)